client/adventureMap/AdventureMapShortcuts.cpp

```diff
@@ -461,7 +469,13 @@ bool AdventureMapShortcuts::optionSidePanelActive()
 	return state == EAdventureState::MAKING_TURN || state == EAdventureState::WORLD_VIEW;
 }
 
+bool AdventureMapShortcuts::optionMapScrollingActive()
+{
+	return state == EAdventureState::MAKING_TURN || state == EAdventureState::WORLD_VIEW || (state == EAdventureState::ENEMY_TURN && isCurrentPlayerHuman());
```

Comment:
I would rather split ENEMY_TURN into ENEMY_AI_TURN and ENEMY_PLAYER_TURN and check accordingly
(feel free to pick better names - e.g. "enemy" here can actually be allied player, it basically "any other player")